image/templates/helm/stackrox-secured-cluster/templates/collector.yaml

```diff
@@ -49,7 +49,11 @@ spec:
         - name: COLLECTOR_CONFIG
           value: '{"tlsConfig":{"caCertPath":"/var/run/secrets/stackrox.io/certs/ca.pem","clientCertPath":"/var/run/secrets/stackrox.io/certs/cert.pem","clientKeyPath":"/var/run/secrets/stackrox.io/certs/key.pem"}}'
         - name: COLLECTION_METHOD
+          {{- if (default ._rox.collector.forceCollectionMethod false) }}
           value: {{ ._rox.collector.collectionMethod }}
+          {{- else }}
+          value: CORE_BPF
+          {{- end }}
```

Comment:
I think that customers who use `NO_COLLECTION` and upgrade will be moved to `CORE_BPF`, despite the PR description saying otherwise.

Also, the behavior of the `forceCollectionMethod` flag in Helm seems to be different to the `ForceCollection` Operator flag, so I want to double check if that was intended. In the operator, it just disables the conversion of `CORE_BPF` to `EBPF`, everything else stays the same. In Helm, it converts everything to `CORE_BPF`.

Reply:
I completely missed
 `{{- if ne ._rox.collector.collectionMethod "NO_COLLECTION"}}`
 as it's not visible in the diff by default.
 
 In this case I think it's fine.